.github/workflows/main.yml

```diff
@@ -12,6 +12,8 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: "13.x"
```

Comment:
I would suggest we use NodeJS 12 everywhere (LTS)

Reply:
Thanks, will change it to 12.x